src/screens/HomeScreen.tsx

```diff
@@ -15,12 +17,38 @@ const styles = StyleSheet.create({
   },
 });
 
+const isValidLocation = (tree: Tree) => {
+  if (tree.location && tree.location.latitude && tree.location.longitude) {
+    return tree;
+  }
+};
+
 export default function HomeScreen() {
   const [trees, setTrees] = useState<Tree[]>([]);
 
   const isValidLocation = (tree: Tree) =>
     tree.location && tree.location.latitude && tree.location.longitude;
 
+  const getCoordinates = (mapboxJSON: Dictionary) => {
```

Comment:
i wouldn't use the Dictionary type here it's not really type safe... do create a type safeguard